drivers/interrupt_controller/intc_plic.c

```diff
@@ -43,7 +43,7 @@
  * However, it is defined and supported by at least the Andes & Telink datasheet, and supported
  * in Linux's SiFive PLIC driver
  */
-#define PLIC_TRIG_LEVEL ((uint32_t)~BIT(0))
+#define PLIC_TRIG_LEVEL ((uint32_t)(0))
 #define PLIC_TRIG_EDGE ((uint32_t)BIT(0))
```

Comment:
Thats an embarassing mistake of mine, do you think if the following would make it even clearer?

```suggestion
#define PLIC_TRIG_LEVEL ((uint32_t)0)
#define PLIC_TRIG_EDGE ((uint32_t)1)
```